src/commands/root.ts

```diff
@@ -21,6 +21,7 @@ export const rootUsage: Usage = () => sprint`
 
     {bold COMMANDS}
       sync           Sync your Gadget application's source code
+      deploy         Deploys your app to production.
```

Comment:
How do you feel about making this like the other descriptions?

```suggestion
      deploy         Deploy your app to production
```